creator-node/docker-compose.yml

```diff
@@ -58,6 +71,8 @@ services:
         mode: non-blocking
         max-buffer-size: 100m
       driver: json-file
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
```

Comment:
Added to both discovery and creator. Used to allow a container to access a host-level port (for the linux exporter) by contacting http://host.docker.internal.

Reply:
How is the linux exporter run?

Reply:
bumping
also worth adding clarifying comment in code for this

Reply:
Updated this thread: https://github.com/AudiusProject/audius-docker-compose/pull/100/files#r993795167 which should have all relevant details.